react/features/speaker-stats/components/SpeakerStats.js

```diff
@@ -0,0 +1,112 @@
+/* global APP */
+
+import React, { Component } from 'react';
+
+import SpeakerStatsItem from './SpeakerStatsItem';
+import SpeakerStatsLabels from './SpeakerStatsLabels';
+
+/**
+ * React component for displaying speaker stats.
+ *
+ * @extends Component
+ */
+class SpeakerStats extends Component {
+    /**
+     * Initializes a new SpeakerStats instance.
+     *
+     * @param {Object} props - The read-only React Component props with which
+     * the new instance is to be initialized.
+     */
+    constructor(props) {
+        super(props);
+        this.updateInterval = null;
+        this.state = {
+            stats: {}
+        };
+    }
+
+    /**
+     * Begin polling for speaker stats updates.
+     *
+     * @inheritdoc
+     * @returns {void}
+     */
+    componentWillMount() {
+        this.fetchUpdate();
+        this.updateInterval = setInterval(this.fetchUpdate.bind(this), 1000);
+    }
+
+    /**
+     * Stop polling for speaker stats updates.
+     *
+     * @inheritdoc
+     * @returns {void}
+     */
+    componentWillUnmount() {
+        clearInterval(this.updateInterval);
+    }
+
+    /**
+     * Get the latest speaker stats.
+     *
+     * @returns {void}
+     */
+    fetchUpdate() {
+        const stats = APP.conference.getSpeakerStats();
```

Comment:
Part of the migration to React includes moving away from the APP global as much as we can. Since this is a new component it should get the conference as a prop, or a way to get it from the Redux store instead.